test/jdk/java/awt/Dialog/DialogInitialResizability.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2003, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Dialog;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.event.ComponentEvent;
+import java.awt.event.ComponentListener;
+
+/*
+ * @test
+ * @bug 4912551
+ * @summary Checks that with resizable set to false before show()
+ *          dialog can not be resized.
+ * @library jdk/java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual DialogInitialResizability
+ */
+
+public class DialogInitialResizability {
+    static String instructions = """
+                    Steps to try to reproduce the problem:
```

Comment:
```suggestion
                    Steps to try to reproduce the problem:
```
Can remove this line.